LibraBFT/Impl/Consensus/SafetyRules/Properties/SafetyRules.agda

```diff
@@ -138,22 +141,22 @@ module ConstructAndSignVoteM where
     -- help ourselves understand.
 
     step₃-contract
-        : ∀ rm pre safetyData voteData ledgerInfo →
-          (RWST-weakestPre (step₃ safetyData voteData author ledgerInfo)
+        : ∀ rm pre safetyData1 voteData ledgerInfo →
```

Comment:
I changed this to `safetyData1` to match the name used in the implementation model.